### ISSUE TITLE
Skin tone selector has insufficient color contrast

### ISSUE THREAD

**Issue by arethsu** [JohnXLivingston/peertube-plugin-livechat#718]
**Description of issue**

This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 1.4.11 Non-text Contrast (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast).

People with reduced vision often have difficulty seeing components and graphics in the interface that do not have high enough contrasts against the background. This includes buttons, form fields, icons, illustrations and diagrams (such as curves and arrows).

In the emoji picker there are options to select which skin tone to use. These options are overlaid on a black background, which causes contrast issues with darker skin tones.

![Contrast checker showing fail of skin color emoji against background, screenshot.](https://github.com/user-attachments/assets/27a62dbc-2f9c-4c81-b27b-004d0298e7b3)

**To Reproduce**

1. Use a contrast checker and measure foreground against background

**Recommended solution**

Graphical objects must have a contrast value of 3:1 against the background. You can use tools such as the the aforementioned https://contrast.report/ to measure this.

Since the range of skin tones here means both light and dark backgrounds are likely to fail sometimes, there may be a need to give each option its own background color. Or, perhaps, an outline that stands out against the background regardless of whether it stands out against the skin tone itself.

**Server:**

 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 14.0.0

**Desktop:**

 - OS: macOS 15.5
 - Browser: Safari 18.5

**Additional context**

This issue was part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), under assignment of [HowlRound Theatre Commons](https://howlround.com/).
